.github/workflows/deploy.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         run: |
           # TODO: create new environment automatically
           # Update/create each deployment file in src/services/kubernetes
-          for f in ./src/services/kubernetes/*.yaml; do
+          for f in $(find ./src/services/kubernetes -type f -name "*.yaml"); do
```

Comment:
This is where the magic happens. There is a job under services/kubernetes/initialise/initialise-job.yaml that is not currently picked up by the existing file glob. This will now traverse into this directory and pick up this file to be applied.